File: JAVA/Human.java
class Human
{
    private String nik;
    private String nama;
    private String jk;

    Human()
    {
        this.nik = "";
        this.nama = "";
        this.jk = "";
    };
    Human(String a, String b, String c)
    {
        this.nik = a;
        this.nama = b;
        this.jk = c;
    }

    void setNik(String a)
    {
        this.nik = a;
    }
    void setNama(String a)
    {
        this.nama = a;
    }
    void setJk(String a)
    {
        this.jk = a;
    }

    String getNik()
    {
        return this.nik;
    }
    String getNama()
    {
        return this.nama;
    }
    String getJk()
    {
        return this.jk;
    }
};

File: PHP/Human.php
<?php
class Human
{
    private $nik = "";
    private $nama = "";
    private $jk = "";
    function __construct($n, $na, $j)
    {
        $this->nik = $n;
        $this->nama = $na;
        $this->jk = $j;
    }
    function setNik($a)
    {
        $this->nik = $a;
    }
    function setNama($a)
    {
        $this->nama = $a;
    }
    function setJk($a)
    {
        $this->jk = $a;
    }

    function getNik()
    {
        return $this->nik;
    }
    function getNama()
    {
        return $this->nama;
    }
    function getJk()
    {
        return $this->jk;
    }
    function __destruct(){
        
    }
}
?>
